pypeit/par/pypeitpar.py

```diff
@@ -1743,6 +1750,10 @@ def __init__(self, bspline_spacing=None, boxcar_radius=None, trace_npoly=None,
         descr['maxnumber'] = 'Maximum number of objects to extract in a science frame.  Use ' \
                              'None for no limit.'
 
+        defaults['find_trim_edge'] = [5,5]
+        dtypes['find_trim_edge'] = list
```

Comment:
Shouldn't this be a tuple? It is not mutable.

Reply:
I tried and failed.  Will try again.

Reply:
sticking with list for now.